devnet/substrate_bob.Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM parity/polkadot:v0.9.10 AS polkadot
+FROM golang:1.17
```

Comment:
It's rather strange to have a golang based image to run polkadot.

We should have instead two builders, one `polkadot` (already one) and one for the datadog yml setup.

Reply:
Nice comment. I changed to:
1. Use the golang image to run the `cmd/update-dd-agent-confd/main.go` made by @timwu20  to generate the `config.yml` file 
2. Use the `parity/polkadot:v0.9.17` to execute the polkadot binary just copying the `config.yml` from the upper golang image

Reply:
Awesome 👍 👍